ui/api/meshmodel.js

```diff
@@ -66,9 +66,9 @@ export async function getVersionedComponentFromModel(
   );
 }
 
-export async function getComponentsDetailWithPageSize(page=1, pageSize="all") {
+export async function getComponentsDetailWithPageSize(page=1, pageSize="all", sortOrder) {
   return await promisifiedDataFetch(
-    `api/meshmodels/components?page=${page}&pagesize=${pageSize}`
+    `api/meshmodels/components?page=${page}&pagesize=${pageSize}&order=${encodeURIComponent(sortOrder)}`
```

Comment:
@UsmanDev09 why are you not using `sort` as a querystring parameter?

From the REST API docs: 
```
?sort={[asc/desc]} Default behavior is asc
```

// @nebula-aac

Reply:
@UsmanDev09 can you finish this now?

Reply:
@leecalcote  Yes, I can